editor/src/clj/editor/pipeline/fontc.clj

```diff
@@ -392,7 +394,11 @@
 (defn- ttf-semi-glyphs [font-desc ^Font font antialias]
   (let [prospect-codepoints (if (:all-chars font-desc)
                               (range 0x10ffff)
-                              (concat (range 32 127) (map int (:extra-characters font-desc))))
+                              (sort
+                                (eduction
+                                  (map int)
+                                  (distinct)
+                                  (:characters font-desc))))
```

Comment:
`eduction` lets us combine several transformations like `(map int)` and `(distinct)` on the sequence `(:characters font-desc)` without generating intermediate collections.